anomaly-benchmarking/generate-datasets/script.whizzml

```diff
@@ -1,3 +1,12 @@
+;; Takes a map of bin labels and max edges, and returns a sorted list
+;; of maps
+
+(define (map-to-list my-map)
```

Comment:
i wouldn't use such an informative name, my-map, for a function parameter (as an aside, i must confess my- prefixes make me cringe, but it you like them i won't object provided they're kept for naming local variables... that said, it's not something i'll ever sanction in a whizzml style guide :-))  here something like labes-to-edges or simply edges or edges-map would be more adequate.